scripts/calculate_trending_scores_daily.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+import _init_path  # noqa: F401 Imported for its side effect of setting PYTHONPATH
+
+from openlibrary.config import load_config
+from openlibrary.plugins.worksearch.code import execute_solr_query
+from openlibrary.plugins.worksearch.search import get_solr
+from scripts.solr_builder.solr_builder.fn_to_cli import FnToCLI
+
+
+def fetch_works(current_day: int):
+    resp = execute_solr_query(
+        '/export',
+        {
+            "q": f'trending_score_hourly_sum:[1 TO *] OR trending_score_daily_{current_day}:[1 TO *]',
+            "fl": "key,trending_score_hourly_sum",
+            "sort": "trending_score_hourly_sum desc",
+        },
+    )
+    doc_data = {}
+    if resp:
+        data = resp.json()
+        try:
+            docs = data["response"]["docs"]
+        except KeyError:
+            raise KeyError
```

Comment:
This is the same since we're raising it again 😁 
```suggestion
        docs = data["response"]["docs"]
```

Reply:
Resolved!